packages/vue/test/base/tests/e2e/specs/inputs.cy.js

```diff
@@ -1,6 +1,6 @@
 describe('Inputs', () => {
   beforeEach(() => {
-    cy.visit('/inputs')
+    cy.visit('/inputs');
```

Comment:
Revert this since this change is not part of the tokens work.
```suggestion
    cy.visit('/inputs')
```